VHAR-4364 Alustataulukko: näytä vähemmän tietoa Toimenpiteen tiedot sarakkeessa

petar-jr opened the pull request:
Älä näyttää otskioa tietyille kentille toimenpide-tiedot sarakkeessa

petar-jr committed:
Älä näyttää otskioa tietyille kentille

jarnovayrynen reviewed (approved):
Alkaapa näyttää hyvältä! Tässä pari pientä korjaustoivetta tikettiin VHAR-4364, Toimenpiteen tiedot sarake:
Verkon osalta poistettava “Verkon tyyppi: teräsverkko”, koska tämä tieto on jo Toimenpide-sarakkeessa
BEST/SST jne: poistettava väliotsikot Sideaine: ja sideainepitoisuus:
KOST: poistettava väliotsikot Sideaine: ja sideainepitoisuus: ja sideaine2, ks. Esimerkki tiketillä: "Käsittelysyvyys 25 cm, Bitumi 20/30 5 %, (UUSIO) CEM I"

Näiden korjausten jälkeen approved ja voit mergetä itse developiin kunhan testit ovat menneet läpi.

petar-jr committed:
Lisää otsikoita skipataan, ja verkon-tyypin poisto

petar-jr committed:
Laskee 'jos' ehdot oikea alustalomakkeessa

petar-jr committed:
Hoita ehdolliset kentät kunnolla

Jos haluat metadata, täytyy lähettää data ja toimenpide, ei
pelkka toimenpide, niin että voi kunnolla laskea kaikki
ehdolliset ('jos') kentät.

petar-jr committed:
Merge branch 'develop' into VHAR-4364-ei-kaikki-otsikot-tarvitse-tiedot-sarakkeessa
